Src/AutoFixtureUnitTest/Kernel/TypeRelayTests.cs

```diff
@@ -171,6 +171,9 @@ public void ShouldFailIfConstructedWithOpenAndNonOpenType(Type from, Type to)
         [InlineData(typeof(IEnumerable<>), typeof(List<>), typeof(IEnumerable<string>), typeof(List<string>))]
         [InlineData(typeof(IReadOnlyDictionary<,>), typeof(IDictionary<,>), typeof(IReadOnlyDictionary<string, byte>), typeof(IDictionary<string, byte>))]
         [InlineData(typeof(Nullable<>), typeof(IEnumerable<>), typeof(int?), typeof(IEnumerable<int>))]
+        [InlineData(typeof(IList<>), typeof(List<>), typeof(IList<string>), typeof(List<string>))]
```

Comment:
Current test verifies the "relaying" capabilities of the `TypeRelay` class only. It doesn't guarantee that some particular relay is registered somewhere. In fact, if you revert change in the `Fixture.cs` file, you'll see that this test still passes.

I'd suggest do not touch this test as it already verifies the similar interfaces and there is no value in verifying each possible pair. Instead, I'd suggest to extend the following test:

https://github.com/AutoFixture/AutoFixture/blob/e9212dc4c527f7446418cefc0340b371bc4cea17/Src/AutoFixtureUnitTest/FixtureTest.cs#L5077

Also it would be cool to add a test at the bottom of the `FixtureTest` which guarantees that if `IReadOnlyList<>` is requested, fixture returns non-`NoSpecimen` instance. Please don't assert the actual result type - it's implementation detail we don't want to guarantee.